trade-finance-manager-ui/component-tests/case/task.component-test.js

```diff
@@ -56,13 +56,13 @@ describe(page, () => {
   });
 
   it('should render deal subheading', () => {
-    const expected = `${params.deal.details.ukefDealId} ${params.deal.submissionDetails.supplierName} > ${params.deal.submissionDetails.buyerName}`;
+    const expected = `View deal${params.deal.details.ukefDealId} details ${params.deal.submissionDetails.supplierName} > ${params.deal.submissionDetails.buyerName}`;
```

Comment:
We need space after `View deal`

Reply:
Added space after '`View deal`' `const expected = `View deal ${params.deal.details.ukefDealId} details ${params.deal.submissionDetails.supplierName} > ${params.deal.submissionDetails.buyerName}`;` for `trade-finance-manager-ui/component-tests/case/task.component-test.js`